examples/dotnet/BUILD

```diff
@@ -0,0 +1,23 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+
+docker_build(
+    name = "hello",
+    base = "//dotnet:dotnet",
+    cmd = [
+        "dotnet",
+        "/bin/Release/netcoreapp2.0/hello.dll",
+    ],
+    files = [":bin"],
```

Comment:
I wouldn't expect this to work unless you check in the `bin/` directory.  Even then, I wouldn't expect adding the directory via `files=[":dirname"]` to work.

Reply:
I do need the contents of the /bin folder to get referenced into the bazel rules.  The bin folder contains the hello.dll file that is generated out of band.   Once the .dll is generated, that file is available for distroless to import into the image itself.

I'm trying this now create an empty folder with a blank file in it

```
//examples/dotnet/bin/README.md
```
where i'll explain in README.md to first run `dotnet restore` and `dotnet publish -c Release` first.

That way, atleast the CI passes (hopefully)